README.md

```diff
@@ -1,36 +1,26 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
-
 ## Getting Started
 
-First, run the development server:
-
-```bash
-npm run dev
-# or
-yarn dev
-```
-
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+## Scripts
 
-You can start editing the page by modifying `pages/index.tsx`. The page auto-updates as you edit the file.
+`postgres:migrate:dev`: this would be used to create and execute migrations on our postgres database.
```

Comment:
These changes were only relevant when the app was fullstack nextjs. With the current changes where the API connection is a Java spring boot backend they're no longer necessary and should removed.

Reply:
Cleaned out the no longer necessary contents.